api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1386,11 +1389,38 @@ public String presignedGetObject(String bucketName, String objectName, Integer e
     String region = BucketRegionCache.INSTANCE.region(bucketName);
 
     Request request = createRequest(Method.GET, bucketName, objectName, region,
-                                    null, null, null, null, 0);
-    HttpUrl url = Signer.presignV4(request, region, accessKey, secretKey, expires);
+                                    null, reqParams, null, null, 0);
+    HttpUrl url = Signer.presignV4(request, region, accessKey, secretKey,
+                                   expires);
```

Comment:
You could avoid indentation here as 150 chars long line is allowed in checkstyle.